cmd/controller/cmd/root.go

```diff
@@ -123,9 +126,11 @@ func executeRootCmd(baseCtx context.Context, cfg *config2.Config) error {
 		limitNamespace = cfg.LimitNamespace
 	}
 	options := manager.Options{
-		Namespace:     limitNamespace,
-		SyncPeriod:    &cfg.DownstreamEval.Duration,
-		ClientBuilder: executors.NewFallbackClientBuilder(propellerScope.NewSubScope("kube")),
+		Namespace:  limitNamespace,
+		SyncPeriod: &cfg.DownstreamEval.Duration,
+		NewClient: func(cache cache.Cache, config *rest.Config, options client.Options, uncachedObjects ...client.Object) (client.Client, error) {
```

Comment:
These are required just because of k8s version updates?

Reply:
yes, correct.